### ISSUE TITLE
Lists of dtypes (with type hints?) not iterable

### ISSUE THREAD

**Issue by ngreenwald** [angelolab/toffy#120]
*Please refer to our [FAQ](https://ark-analysis.readthedocs.io/en/latest/_rtd/faq.html) and look at our [known issues](https://github.com/angelolab/ark-analysis/issues) before opening a bug report.*

**Describe the bug**
The modified verify_in_list functionality is failing for the tiling checks in toffy related to dtypes

**To Reproduce**
![image](https://user-images.githubusercontent.com/13770365/172065687-457f6778-0ac7-4ccd-afa3-65f68d9a10c4.png)

**Comment by ngreenwald** [comment]
this will need to be addressed before we can move to the newest version of ark

**Comment by srivarra** [comment]
I have addressed this in https://github.com/angelolab/ark-analysis/pull/592/commits/f9da24a21cec8f8c383788201af9b1af5a6f2d37. I will copy the contents over into a new PR for it.
